### DISCUSSION TITLE
Running xformers-rocm on a RX7900XTX in ubuntu 22.04.05, What is your setup?

### DISCUSSION THREAD

**Discussion by spumabeat** [ROCm/ROCm#3869] [Q&A] [answered]
I am having problems making this library run, I am using ROCM 6.2 as my driver and trying to run it with pytorch 2.4.1 but is no way it runs, do you have the correct config for sharing with us the community?

Love and respect to all!

**Comment by zichguan-amd** [comment, marked as answer]
Which version of xformers are you trying to use? You can follow the installation guide here https://github.com/facebookresearch/xformers for the latest version (0.0.28.post3) which requires PyTorch 2.5.1. If you want to use ROCm 6.2, you can get PyTorch 2.5.1 with `pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm6.2` and use the xformers wheels for ROCm 6.1 `pip3 install -U xformers --index-url https://download.pytorch.org/whl/rocm6.1` or build from source against ROCm 6.2.
